DESCRIPTION

```diff
@@ -29,10 +29,10 @@ Imports:
     rversions,
     stats,
     git2r (>= 0.10.1)
-Suggests: 
+Suggests:
     testthat (>= 0.7),
     BiocInstaller,
-    Rcpp (>= 0.10.0),
+    Rcpp (>= 0.11.0),
```

Comment:
Do you realise that you accidentally modified the version of Rcpp instead of git2r?